assets/src/dashboard/components/infiniteScroller/index.js

```diff
@@ -40,18 +41,14 @@ const InfiniteScroller = ({
   const [loadMore, setLoadMore] = useState(false);
   const loadingRef = useRef(null);
   const containerRef = useRef(null);
+
   useEffect(() => {
     if (loadMore) {
       handleGetData(loadMore);
-      setLoadMore(false);
     }
+    setLoadMore(false);
```

Comment:
this seems like a separate effect that should fire off something like:
```
useEffect(() => {
  if (!isLoading) {
    dispatch(canLoadMore ? 'LOADABLE' : 'COMPLETE');
  }
}, [isLoading, canLoadMore]);
```

Reply:
We could add a separate effect that sets this into a loading state if `Typeahead` were to initiate a loading state externally to this component. Might make us alter our state machine a little bit here tho because the implementation I recommended has an effect that fires the load event it's responsible for on load. 

If you decide to implement this with a state machine I'd be happy to pair on this with you if you'd like